tests/phpunit/integration/Modules/Reader_Revenue_Manager/Web_TagTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\Modules\Reader_Revenue_Manager\Web_TagTest
+ *
+ * @package   Google\Site_Kit\Tests\Modules\Reader_Revenue_Manager
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Modules\Reader_Revenue_Manager;
+
+use Google\Site_Kit\Modules\Reader_Revenue_Manager;
+use Google\Site_Kit\Modules\Reader_Revenue_Manager\Web_Tag;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Modules
+ * @group Reader_Revenue_Manager
+ */
+class Web_TagTest extends TestCase {
+	const PUBLICATION_ID = '12345';
+
+	public function test_register() {
+		$web_tag = new Web_Tag( self::PUBLICATION_ID, Reader_Revenue_Manager::MODULE_SLUG );
+		$web_tag->register();
+
+		do_action( 'wp_enqueue_scripts' );
+
+		$footer_html = $this->capture_action( 'wp_footer' );
+
+		$this->assertStringContainsString( 'Google Reader Revenue Manager snippet added by Site Kit', $footer_html );
+		$this->assertStringContainsString( 'script type="text/javascript" src="https://news.google.com/swg/js/v1/swg-basic.js" id="google_swgjs-js"></script>', $footer_html );
```

Comment:
Super trivial, but it would be nice to include the opening bracket for this test, similar to other assertions elsewhere in the codebase.

```suggestion
		$this->assertStringContainsString( '<script type="text/javascript" src="https://news.google.com/swg/js/v1/swg-basic.js" id="google_swgjs-js"></script>', $footer_html ); // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```

Reply:
This was an oversight, updated, thank you!